packages/suite/src/storage/migrations/index.ts

```diff
@@ -1176,4 +1178,56 @@ export const migrate: OnUpgradeFunc<SuiteDBSchema> = async (
             return account;
         });
     }
+
+    // Deprecate Vertcoin (VTC) and other networks
+    if (oldVersion < 52) {
+        const deprecatedNetworks = ['vtc', 'btg', 'nmc', 'dgb', 'dash'];
+
+        // Remove transactions related to deprecated networks
+        await updateAll(transaction, 'txs', tx => {
```

Comment:
I think you miss `graph`, `historicRates` and `sendFormDrafts`

Reply:
This is true, but historically we didn't handle these when removing coins, only when changing symbols (because we wanted to keep the data), so I believe this should be safe as well.